Sources/UtiliKit/ScrollingPageControl/ScrollingPageControl.swift

```diff
@@ -229,7 +231,8 @@ class ScrollingPageControl: UIView {
     // MARK: UIView
     override var intrinsicContentSize: CGSize {
         let visibleDots = CGFloat(min(numberOfPages, maxVisibleDots))
-        return CGSize(width: (visibleDots * dotSize.width) + ((visibleDots - 1.0) * dotSpacing), height: max(37.0, dotSize.height))
+        let minimumControlHeight:CGFloat = 37.0
```

Comment:
Very minor - can we get a space after the colon here?
```suggestion
        let minimumControlHeight: CGFloat = 37.0
```